src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -750,7 +750,7 @@ void StorageReplicatedMergeTree::drop()
     {
         /// Table can be shut down, restarting thread is not active
         /// and calling StorageReplicatedMergeTree::getZooKeeper() won't suffice.
-        auto zookeeper = global_context.getZooKeeper();
+        auto zookeeper = getZooKeeper();
```

Comment:
The comment above says "and calling StorageReplicatedMergeTree::getZooKeeper() won't suffice", does it not apply anymore?

Reply:
@cyberhuman, Thanks. The comment is still correct now. I'll fix it.